source/slang/slang-ir-reachability.h

```diff
@@ -16,23 +16,8 @@ struct ReachabilityContext
     ReachabilityContext() = default;
     ReachabilityContext(IRGlobalValueWithCode* code);
 
+    bool isInstReachable(IRInst* from, IRInst* to);
```

Comment:
We can revert this change and move `isInstReachable` to its original place?

Reply:
We can but after switching to `getBlock` calls we would also need to `#include "slang-ir-util.h"` in `slang-ir-reachabilty.h` is that fine?

Reply:
Ah, I see, moving it here is fine.